tests/test_send_stats_to_performance_platform.py

```diff
@@ -0,0 +1,359 @@
+import json
```

Comment:
Correct me if I'm wrong but `send_data` isn't itself tested at any point. Oversight or did you think it's too simplistic to bother?